### ISSUE TITLE
Risk of reputational harm to organizations displaying inaccurate accessibility metadata

### ISSUE THREAD

**Issue by avneeshsingh** [w3c/publ-a11y#402]
This issues is submitted by Mark Weiler, Phd. - Wilfrid Laurier University.

As a Web and User Experience librarian at a university, I have concerns with the guidance. With regards to the specific techniques for displaying meta-data, the guide simply suggests following EPUB and ONIX techniques. I believe this UX guide should not defer responsibility this way. It should invest more thought in considering  how to display this information.

For example, while both the EPUB instructions and ONIX instructions for displaying metadata recommend displaying the certifier, if it exists, they do not provide any guidance on what to display when there is no certifier or no certifier credentials.  Drawing on these techniques means university libraries are being advised to display uncertified books with no indicator that they are uncertified.  Users and the general community may not know where to focus their feelings when they experience problems with uncertified books.  For this reason, the UX guide should go beyond the EPUB and ONIX techniques and state: If the book has no certifier, then display a message “Not certified”. If the certifier does not have credentials, then display a message “Not credentialed”.

**Comment by HadrienGardeur** [comment]
This feels like a very slippery slope to follow since certification won't be required in many countries.

I think that individual organizations such as a university libraries are free to make this kind of choice, but they can hardly be enforced globally.

**Comment by clapierre** [comment]
> This feels like a very slippery slope to follow since certification won't be required in many countries.

Except that if there is a conformsTo statement you MUST include who certified this as indicated by the accessibility 1.1 specification. It can be the publisher i.e. self-certified, or a third-party certifier who may or maynot have a certifierCredential.

I agree with what Mark suggests that it might be a good idea to state that there is no certifierCredential.  Now if there is a conformsTo without a certifiedBy which is in violation of the standard itself what should we do in this case? Pointing this out would be a good idea, that this is in violation of the specification. Extreme position would be not to display the conformance statement, as we have no idea who is claiming this statement.

**Comment by wareid** [comment]
It's only a violation of EPUB Accessibility 1.1, not a violation of WCAG, ONIX, or any other related standard. We've opted to not show certain fields when there is no data for them (Certifier/Certifier credentials for example), because not having information is almost more confusing than having it. I think this part of the UX guidelines can fall to the discretion of the implementer. This is also a worthwhile area to keep an eye on, it's hard to know currently what weight readers will put on credentials/certification, if we see a lot of interest or demand for it, that could influence how this is updated. 

Certification is not required in most places, and there is not necessarily a credentialing authority everywhere even if a publisher wanted certification, and I think it's unclear in the UX Guide that publishers can self-certify. I know this is mentioned in EPUB A11y 1.1, but it may also be worth mentioning in the guidelines as well.

**Comment by clapierre** [comment]
The group agreed to display this metadata as published.
metadata and reality can differ.
If there are false claims there will be a natural selection process of what not to trust.
